src/components/campaigns/CampaignSelect.tsx

```diff
@@ -31,8 +33,9 @@ export default function CampaignSelect({
 
   const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
```

Comment:
If you change the typing like that you won't need the cast below
```suggestion
  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Reply:
fixed